postgresql/provider.go

```diff
@@ -90,6 +92,13 @@ func Provider() *schema.Provider {
 				Description: "AWS region to use for IAM auth",
 			},
 
+			"aws_rds_iam_provider_role_arn": {
```

Comment:
Maybe we could get rid of the `provider` part? Or do you find it not clear enough without it?
```suggestion
			"aws_rds_iam_role_arn": {
```

Reply:
I actually thought about this, and chose to include the provider part because each RDS database has an IAM role ARN, and I wanted to stress that we are not referring to that. I'm OK with either though.

Reply:
I'll resolve this without changing the name for now - do let me know if you insist on the change.